client/me/billing-history/transactions-table.jsx

```diff
@@ -13,11 +13,14 @@ import { capitalPDangit } from 'lib/formatting';
 /**
  * Internal dependencies
  */
+import CompactCard from 'components/card/compact';
+import Pagination from 'components/pagination';
 import TransactionsHeader from './transactions-header';
 import tableRows from './table-rows';
-
 import SearchCard from 'components/search-card';
 
+const PAGE_SIZE = 10;
```

Comment:
I wonder if making the page size a little smaller will help until we get these into separate views.